Add model and command

m0533199321 opened the pull request.

Nanif commented on FFmpeg.Infrastructure/Services/FFmpegServiceFactory.cs:
```diff
@@ -42,5 +42,6 @@ public ICommand<TimestampModel> CreateTimestampCommand()
         {
             return new TimestampCommand(_executor, _commandBuilder);
         }
+
```
Unnecessary blank line – consider removing it for cleaner formatting.

m0533199321 replied:
I changed these changes in the second pull request.
They were in the departments I uploaded in that one, not in this one.

Nanif commented on Ffmpeg.API/Program.cs:
```diff
@@ -56,6 +56,8 @@
 app.MapEndpoints();
 
+
+
```
Unnecessary blank lines – consider removing it for cleaner formatting.

m0533199321 replied:
I changed these changes in the second pull request.
They were in the departments I uploaded in that one, not in this one.

Nanif commented:
Please address the comments first, and only then assign me as an assignee.

m0533199321 commented:
סליחה שאני משגעת אותך,
הסתבכתי מכל המיילים כבר.
אם אין את הPR שלי ברשימה אז המשימה הושלמה?

תודה על הכל!!

‫בתאריך יום א׳, 27 ביולי 2025 ב-15:27 מאת מלכה הילדסהיימר <‪
***@***.***‬‏>:‬

> *I’ve addressed and updated all the comments you mentioned. Let me know if
> there’s anything else needed*
>
>
> ‫בתאריך יום א׳, 27 ביולי 2025 ב-15:25 מאת ‪Neomi Fishbein‬‏ <‪
> ***@***.***‬‏>:‬
>
>> *Nanif* left a comment (wolf-rnd-org/PrePracticum2#16)
>> <https://github.com/wolf-rnd-org/PrePracticum2/pull/16#issuecomment-3124371337>
>>
>> Please address the comments first, and only then assign me as an assignee.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/wolf-rnd-org/PrePracticum2/pull/16#issuecomment-3124371337>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BLR2OUXNSPEFLIN6TUJV2VT3KTAMHAVCNFSM6AAAAACCOTWBQ6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTCMRUGM3TCMZTG4>
>> .
>> You are receiving this because you were assigned.Message ID:
>> ***@***.***>
>>
>
